### ISSUE TITLE
Domain now points to domain registrar - maybe it wasn't renewed?

### ISSUE THREAD

**Issue by m-col** [qtile/qtile.org#41]
Currently qtile.org looks like this:
![image](https://user-images.githubusercontent.com/44536113/171421383-d11ff156-60e0-4218-aa2a-4d26c1fbf102.png)

So it looks like we need to re-register. To avoid being presumptuous I'll put it out there that I am more than happy to cover the cost of registration.

**Comment by tych0** [comment]
Yeah, I'm happy to fund it too. I emailed @cortesi, he's been paying since inception. We'll see what he says.

**Comment by Lord-Valen** [comment]
> Hey folks. Apologies for the disruption - I've just renewed the domain for 5 years, and resolution now works. Oddly, the domain was not due to expire for another 2 weeks - I'll dig a bit more to figure out what was going on. 

That's odd, because qtile.org still points to Pair.

**Comment by tych0** [comment]
It does if you use https://qtile.org, but if you use http://qtile.org, you get the right thing. Seems bad that they're hijacking... but shouldn't happen unless people are using https-everywhere style extensions.

**Comment by pahakalle** [comment]
The dns A record for qtile.org does not seem top be pointing to heroku, but to the registrat. www.qtile.org is pointing to heroku. Can the qtile.org A record be editet to point to heroku or make it point to the www. record.

**Comment by elParaguayo** [comment]
@tych0 not sure the domain stuff is sorted yet, we're still not getting SSL on the website (people muttering on Reddit...)

**Comment by tych0** [comment]
Yeah, it's not. Been having a problem getting through @cortesi 's e-mail filters. Maybe someone can ping him on twitter?

**Comment by elParaguayo** [comment]
I'm not on Twitter. Could any of @ramnes @m-col or @jwijenbergh help here?

**Comment by ramnes** [comment]
Aldo is a hard man to reach

<img width="185" alt="image" src="https://github.com/qtile/qtile.org/assets/835072/05e6eae7-b7e5-4ba4-9874-5ca11237829c">

**Comment by ramnes** [comment]
That was a Twitter DM

**Comment by cortesi** [comment]
@ramnes Hmm - odd. My DMs are open, and I have recent messages from people. This may just be Twitter flakiness, as opposed to Cortesi flakiness.

**Comment by cortesi** [comment]
Folks, bear with me for a bit. I am transferring the qtile domain from Pair to AWS to make management easier. We should be back up and running shortly.

**Comment by elParaguayo** [comment]
Not sure if you've manage to make any changes but it doesn't seem to be working yet.

**Comment by cortesi** [comment]
Folks, the domain transfer completed about an hour ago. I've set up the Github pages domain and changed the name severs over to AWS. We're now waiting for everything to propagate so Github can validate the configuration, then we should be up and running. I'm keeping an eye on things, and will close this ticket once everything is up.

**Comment by elParaguayo** [comment]
Thanks @cortesi 

From what I can see:
- The domain points correctly for http but https isn't working (presumably we need to configure that separately).
- https://docs.qtile.org is not pointing to readthedocs (same for plain http).

**Comment by tych0** [comment]
Seems like https is working for qtile.org now at least for me. Once the docs page is redirected I think we can finally close this!

**Comment by elParaguayo** [comment]
I'm still getting an insecure warning:
```
Firefox does not trust this site because it uses a certificate that is not valid for www.qtile.org. The certificate is only valid for the following names: *.github.io, github.io, *.github.com, github.com, www.github.com, *.githubusercontent.com, githubusercontent.com
```

**Comment by tych0** [comment]
Yeah, you're right. I probably saved github's cert, whoops. I don't see any cnames for qtile.org either:

```
~/packages/qtile in-line dig +short qtile.org A
185.199.108.153
185.199.111.153
185.199.109.153
185.199.110.153
~/packages/qtile in-line dig +short qtile.org CNAME
~/packages/qtile in-line
```

**Comment by ramnes** [comment]
@cortesi We have bunch of people reporting that the docs are down. If that's quicker and that you feel like it, feel free to add me to AWS and readthedocs so that I can fix it. :)

**Comment by 10leej** [comment]
So docs.qtile.org is still not resolving and it seems qtile.org is setup with github properly.  

What can I do to help resolve this?

**Comment by ramnes** [comment]
Not much, we're waiting for Aldo, he's a SPOF here :(

**Comment by jwijenbergh** [comment]
https now works on qtile.org and is enforced (just had to re-apply for the DNS check). docs DNS entry still need to be set up.

Also https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages should be followed.

Friendly ping @cortesi

**Comment by ramnes** [comment]
@jwijenbergh HTTPS is down again following the adventures with #49 and #50. What did you do to fix it exactly? Did you delete and create the CNAME again in GitHub settings? If so, I tried it already and it doesn't seem to help, maybe we need to wait a bit and try again in 24h or something.

**Comment by elParaguayo** [comment]
Hosting on GitHub pages really should work.

There must be something we're missing.

**Comment by tych0** [comment]
It does @ramnes just fixed it. See git HEAD

**Comment by jwijenbergh** [comment]
> It does @ramnes just fixed it. See git HEAD

It's still broken for me

**Comment by ramnes** [comment]
Yeah GitHub hosting works, but their cert management smells. Let's give GitHub a day or two before we try again, I did a lot of trials today and there's probably a rate limit somewhere.

**Comment by ramnes** [comment]
Bingo! We now have a valid certificate but for some reason I can't enforce HTTPS. I'll try again in a few minutes and if that still doesn't work, let's wait another 24h, lol.

**Comment by jwijenbergh** [comment]
Ping @cortesi 

![image](https://github.com/qtile/qtile.org/assets/46386452/932c4e84-fe38-4221-865c-eaad640a5d61)

**Comment by elParaguayo** [comment]
@cortesi Can you please help us address the docs.qtile.org issue?

Ideally, it would be great if you could give someone else in the team access to the domain records so we can move a bit quicker in the future.

Failing that: @m-col @ramnes @tych0 @jwijenbergh we could register a new domain name e.g. qtilewm.org

**Comment by ramnes** [comment]
Fixed apparently
